### PR TITLE
Update enable-attack-surface-reduction.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line about selecting settings catalog if there are any ASR rules already  set via Endpoint Security

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7721

File: microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ You can use Microsoft Endpoint Manager (MEM) OMA-URI to configure custom ASR rul
 
    - In **Platform**, select **Windows 10 and later**
    - In **Profile type**, select **Templates**
+   - If there are ASR rules already set through endpoint securituy, In **Profile type**, select **Settings Catolog**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md]:
```suggestion
   - If ASR rules are already set through Endpoint security, in **Profile type**, select **Settings Catalog**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by jweston-1** [comment]
Approved.

**Pull request merged**